### PR TITLE
[circle2circle] Dredd test for fuse Add/Mul to FC and fold Mul

### PR DESCRIPTION
**Pull request by seanshpark**
This will add circle2circle dredd test for fuse Add/Mul to FC and fold Mul.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for #13411
- from #13415

File: compiler/circle2circle-dredd-recipe-test/test.lst [hunk 1]
```diff
@@ -61,6 +64,9 @@ Add(Net_Mul_Add_002 PASS remove_unnecessary_add)
 Add(Net_Mul_Add_003 PASS remove_unnecessary_add)
 Add(Net_Mul_Div_000 PASS fuse_mul_with_div)
 Add(Net_Mul_Div_001 PASS fuse_mul_with_div)
+Add(Net_Mul_FullyConnected_000 PASS fuse_mul_to_fullyconnected_weights fold_mul)
+Add(Net_Mul_FullyConnected_001 PASS fuse_mul_to_fullyconnected_weights fold_mul)
+Add(Net_Mul_FullyConnected_002 PASS fuse_mul_to_fullyconnected_weights fold_mul)
```
Comment (shs-park) [inline compiler/circle2circle-dredd-recipe-test/test.lst:69]: Unused parameters..!?

```suggestion
Add(Net_Mul_FullyConnected_000 PASS fuse_mul_to_fullyconnected_weights)
Add(Net_Mul_FullyConnected_001 PASS fuse_mul_to_fullyconnected_weights)
Add(Net_Mul_FullyConnected_002 PASS fuse_mul_to_fullyconnected_weights)
```

**Review by shs-park — COMMENTED** [review]
I left a comment, PTAL
=)

File: compiler/circle2circle-dredd-recipe-test/test.lst (hunk 1 above)
Comment (seanshpark) [reply compiler/circle2circle-dredd-recipe-test/test.lst:69]: `fold_mul` is an option needed to constant fold `Mul` op,
- this model has `FC(Mul(input,scale), weights, bias)`.
- `fuse_mul_to_fullyconnected_weights` will convert to `FC(input, Mul(weights, scale), bias)`
- dredd test cannot detect the difference as there exist same number of FC and Mul
- `fold_mul` will convert to `FC(input, weights', bias)` and we can detect absence of `Mul`

Comment (shs-park) [reply compiler/circle2circle-dredd-recipe-test/test.lst:69]: I understand.
Thank you for the description =)

**Review by shs-park — APPROVED** [review]
LGTM!
=)

**Pull request merged**